Week3Project/Week3Project/users.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Week3Project
+{
+    class User
+    {
+        public string firstName { get; set; }
+        public string lastName { get; set; }
+        public int favoriteNumber { get; set; }
+        public int numberOfPets { get; set; }
+        public string namesOfPets { get; set; }
+
+        public string AnswerQuestion(string question)
+        {
+            Console.WriteLine(question);
+            return Console.ReadLine();
+
+        }
+        public int AnswerQuestionWithInteger(string question)
```

Comment:
This is subjective, and some people don't, but I would recommend having a blank line between the closing curly brace of the previous method (`AnswerQuestion`) and this one (between lines 20 and 21)